proposals/lip-0051.md

```diff
@@ -127,7 +127,7 @@ The following constants are used throughout the document:
 | `MODULE_NAME_TOKEN` | string | "token" |
 | `COMMAND_NAME_TRANSFER` | string | "transfer" |
 | `COMMAND_NAME_CROSS_CHAIN_TRANSFER` | string | "transferCrossChain" |
-| `CROSS_CHAIN_COMMAND_NAME_TRANSFER` | string | TBD |
+| `CROSS_CHAIN_COMMAND_NAME_TRANSFER` | string | “crossChainTransferCommand” |
```

Comment:
In the implementation I see they used a different value: ''transferCrossChain", see [here](https://github.com/LiskHQ/lisk-sdk/blob/0f1296c62585f26f3291f3e1a01aa757801f93f0/framework/src/modules/token/constants.ts#L15). 

We should also make sure that for commands we have the same values as in the implementation. If not, we should follow their naming conventions for simplicity.

Reply:
Note that "transferCrossChain" is also the value of  `COMMAND_NAME_CROSS_CHAIN_TRANSFER`. I suggest using the string proposed by Miroslav to avoid confusion (also update the SDK value).

Reply:
Ah, I used that name because of [this line](https://github.com/LiskHQ/lisk-sdk/blob/0f1296c62585f26f3291f3e1a01aa757801f93f0/framework/src/modules/token/module.ts#L83). What should it be (@sergeyshemyakov)?

Reply:
I would say `CROSS_CHAIN_COMMAND_NAME_TRANSFER` should stay "crossChainTransferCommand" and the SDK should be notified so they make sure they align with the LIP.

Reply:
@sitetester I wouldn't say we should change the name of the constant, we're debating about it's _value_

Reply:
The name of the cross-chain command is also "transferCrossChain" in the implementation so I updated it accordingly. From the SDK team they see now issue with confusions and I think we should then follow their decision for the LIP.